project/Dependencies.scala

```diff
@@ -20,15 +21,15 @@ object Dependencies {
     "specs2-core",
     "specs2-junit",
     "specs2-mock"
-  ).map("org.specs2" %% _ % specsVersion)
+  ).map("org.specs2" %% _ % specsVersion cross CrossVersion.for3Use2_13)
```

Comment:
specs2 has v5 in RC phase, which gets build for Scala 3: https://github.com/etorreborre/specs2/tree/SPECS2-5.0.0-RC-05
Shouldn't we make use of that?

Reply:
specs2-mock 5.x does not available and 5.x break some some compatibility.